pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -228,6 +228,23 @@ func (m *manager) reconcileState() (success []reconciledContainer, failure []rec
 				continue
 			}
 
+			// Check whether container is present in state, there may be 2 reasons why it's not present
+			// either container is Be/Bu or kubelet just restarted
```

Comment:
Add 3rd reason here (per your note in the PR comments) which is the container might be in the process of being destroyed. 

> Additional note:
> During pod deletion, RemoveContainer is called, then reconcile loop kicks is, finds the deleted container and because it is not present in state, default cpuset is assigned during container termination process.

> In added check if that case is found (pod is running but DeletionTimestamp is present) container cpuset update is skipped

Reply:
ok